src/Web/Startup.cs

```diff
@@ -43,10 +43,10 @@ public Startup(IConfiguration configuration)
         public void ConfigureDevelopmentServices(IServiceCollection services)
         {
             // use in-memory database
-            ConfigureInMemoryDatabases(services);
+            // ConfigureInMemoryDatabases(services);
```

Comment:
Do we want to keep this change? Would using the in-memory database be preferred by default?